__tests__/app.test.js

```diff
@@ -111,4 +111,50 @@ describe("/api/articles/:article_id", () => {
       })
     })
   });
-});
\ No newline at end of file
+});
+describe("/api/articles/:article_id/comments", () => {
+  describe("GET", () => {
+    test("200: Responds with a array of all comments when passed in the article_id", () => {
+      return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments.length).toBeGreaterThan(0);
+        body.comments.forEach((comment) => {
+          expect.objectContaining({
+            article_id: 1,
+            comment_id: expect.any(Number),
+            author: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            body: expect.any(String),
+          })
+        })
+      })
+    })
```

Comment:
We need to make sure the comments are coming back with the most recent first. Could we just add something in here please to check this?